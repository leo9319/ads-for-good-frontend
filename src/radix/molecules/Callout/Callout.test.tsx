import React from 'react';
import { render } from '@testing-library/react';
import CalloutMolecule from './Callout';

describe('CalloutMolecule', () => {
  it('should render successfully', () => {
    const { baseElement } = render(
      <CalloutMolecule
        iconName="icon-info"
        alternateCloseIcon="icon-close"
        title="Your subscription has expired"
        type="info"
        isVisible
      >
        Vitae adipiscing venenatis sodales sed accumsan cursus augue ornare ut.
        Ut. Sed quis integer nec est. Efficitur risus orci, in platea sodales
        ut.
      </CalloutMolecule>
    );
    expect(baseElement).toMatchSnapshot();
  });
});
