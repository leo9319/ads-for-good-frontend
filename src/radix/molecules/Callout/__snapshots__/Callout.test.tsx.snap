// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CalloutMolecule should render successfully 1`] = `
<body>
  <div>
    <div
      class="rt-CalloutRoot rt-r-size-2 rt-variant-soft rt-Flex rt-r-ai-stretch rt-r-jc-center calloutRoot info"
      data-accent-color=""
    >
      <div
        class="rt-Box callout"
      >
        <div
          class="rt-Box contentbox"
        >
          <p
            class="rt-Text rt-r-size-2 rt-CalloutText title body-md typography-font-weight-semibold"
          >
            Your subscription has expired
          </p>
          <div
            class="rt-Box body-sm typography-font-weight-regular"
          >
            Vitae adipiscing venenatis sodales sed accumsan cursus augue ornare ut. Ut. Sed quis integer nec est. Efficitur risus orci, in platea sodales ut.
          </div>
        </div>
      </div>
    </div>
  </div>
</body>
`;
